src/pages/Environments.js

```diff
@@ -426,16 +429,23 @@ const Environments = () => {
             headerRenderer: () => h(Sortable, { sort: diskSort, field: 'project', onSort: setDiskSort }, ['Billing project']),
             cellRenderer: ({ rowIndex }) => {
               const disk = filteredDisks[rowIndex]
-              const galaxyDiskNames = _.map(disk => disk.name, galaxyDisks)
-              const runtimeDisks = _.remove(disk => _.includes(disk.name, galaxyDiskNames), disks)
-              const runtimeDiskNames = _.map(disk => disk.name, runtimeDisks)
-              const multipleRuntimeDisks = _.remove(disk => _.includes(disk.name, galaxyDiskNames) || disk.status === 'Deleting',
-                disksByProject[disk.googleProject]).length > 1
-              const multipleGalaxyDisks = _.remove(disk => _.includes(disk.name, runtimeDiskNames) || disk.status === 'DELETING',
+              const appDiskNames = _.map(disk => disk.name, appDisks)
+              const multipleRuntimeDisks = _.remove(disk => _.includes(disk.name, appDiskNames) || disk.status === 'Deleting',
                 disksByProject[disk.googleProject]).length > 1
+              const getAppType = diskName => {
+                // appDisks has appType populated on it if the disk was related to an app, but
+                // disksByProject (which is populated from disks) does not.
+                return _.find(disk => disk.name === diskName)(appDisks)?.appType
```

Comment:
In general, we like to avoid the double-call when possible:
```suggestion
                return _.find({ name: diskName }, appDisks)?.appType
```